### ISSUE TITLE
request for source archive on sourceforge

### ISSUE THREAD

**Issue by ryan-sg** [0ldsk00l/nestopia#376]
Hi!  I was wondering if you could put a nestopia-1.51.1.tgz source archive over here: https://sourceforge.net/projects/nestopiaue/files/1.51/

Rationale is: while the github archive links are convenient in a pinch, they can mysteriously change checksum values between fetchings, making packaging systems downstream scoff at changed distfiles.

Thank you for continuing work on this amazing emulator!

**Comment by rdanbrook** [comment]
Yeah, I can definitely do this, will try to get it done over the weekend.

Just curious, what distro or packaging system is it you're working with?

**Comment by ryan-sg** [comment]
Thanks!  I'm working with OpenBSD packages.  Currently, nestopia 1.49 is in the ports tree there.  

On a side note, I noticed it looks like the nestopia tree has been split between nestopia 'proper' (this one) and nestopia-1 for the libretro stuff.

The package maintainer (@bentley) has done a neat job of using the MULTI_PACKAGES facilities of OpenBSD ports to generate both nestopia-1.49 and libretro-nestopia-1.49 packages from the single source archive.  Currently in discussion with him while working on a patch to get the 1.51.1 version in there to see how we might handle that going forward.

TLDR; I might also end up with a request on your other tree for something like a nestopia-libretro-1.51.1.tgz source archive as well :smile:

**Comment by rdanbrook** [comment]
I just managed to build up the energy to clone the repo to /tmp and tar it up, then upload it :).

Glad to hear OpenBSD is going to have an updated version. I've always been a fan of that OS.

For libretro, it's best to use libretro's repository. I sometimes work on that port, but the libretro project often adds their own stuff and it's easier for everyone if they manage the port with me as an occasional contributor to bring in bugfixes and updates from upstream. The libretro port, and most libretro ports, are not really versioned, it is more of a rolling release system, so you'll probably need a different strategy for that.

**Comment by ryan-sg** [comment]
Cool, that was fast :)

As the libretro side of things needs some thinking, I'll close this issue off.  Thank you!
